### PR TITLE
Added book tests

### PR DESCRIPTION
**Pull request by annton-shakur**

### CODE REVIEW

**Review by fedorovychh — APPROVED** [review]
GJ!

File: src/test/java/mate/academy/onlinebookstoreproject/controller/BookControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+        flowersForAlgernonDto.setDescription("A science fiction novel");
+        flowersForAlgernonDto.setCoverImage("flowers-for-algernon.jpg");
+        flowersForAlgernonDto.setCategoryIds(Set.of(1L));
+        flowersForAlgernonDto.setIsbn("978-3-1136-5552-7");
+    }
+
+    @WithMockUser(username = "user")
+    @Test
+    @DisplayName("Get a list of 2 books mapped to dto from the DB ")
+    void getAll_Return_TwoBookDto() throws Exception {
```
Comment (vovasalyha) [inline src/test/java/mate/academy/onlinebookstoreproject/controller/BookControllerTest.java]: This name is not fully correct. Please check some guidance. ([like this](https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea)) It should have a pattern:  MethodName_StateUnderTest_ExpectedBehavior

File: src/test/java/mate/academy/onlinebookstoreproject/controller/BookControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+        expected.add(theGreatGatsbyBookDto);
+        expected.add(vanityFairBookDto);
+        BookDto[] actual = objectMapper.readValue(
+                result.getResponse().getContentAsByteArray(), BookDto[].class);
+        Assertions.assertIterableEquals(expected, Arrays.asList(actual));
+    }
+
+    @WithMockUser(username = "user")
+    @Test
+    void findById() throws Exception {
```
Comment (vovasalyha) [inline src/test/java/mate/academy/onlinebookstoreproject/controller/BookControllerTest.java]: Bad method name

**Review by vovasalyha — APPROVED** [review]
Approving, but please pat attention to method names.

**Pull request merged**